Form-Controls/index.html

```diff
@@ -13,15 +13,44 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+          <fieldset>
+            <legend>Confirm Personal Details</legend>
+            <label for="name">First Name</label><br />
+            <input type="text" id="name" name="name" required minlength="2" placeholder="John"><br />
+  
+            <label for="email">Email</label><br />
+            <input type="email" id="email" name="email" required><br />
+          </fieldset>
+  
+          <fieldset>
+            <legend>Choices for the Item</legend>
+  
+            <label for="chosen_color">Color</label><br />
+            <select id="chosen_color" name="chosen_color" required>
+              <option value="" disabled selected>Select a color</option>
```

Comment:
👍 Good job giving the user a default value rather than defaulting to the first value in the list of options.